src/index.py

```diff
@@ -48,17 +48,20 @@ def handler(event, lambda_context):
         result = TransformationResult.Ok
 
     except CallThrottlingException:
-        log_multiline_message("Call Throttling Exception, Kinesis batch will be marked as OK, but some data is dropped")
+        log_multiline_message("Call Throttling Exception, Kinesis batch will be marked as OK, but some data is dropped",
+                              os.path.basename(__file__) + "_" + handler.__name__ + "_1")
```

Comment:
I would just put some string constant as 2nd param in all usages. Like "call-throttling-exception" etc
What do you think @dt-be @wojtasskorcz ?

Reply:
It looks a bit messy this way for me

Reply:
Hmm, this would require coming up with descriptive strings and I can see potential name clashes. What we need here is a mechanism to generate unique strings easily, and copy-pasting the same line all over the codebase seems easy enough. But I can see your point that it's not the easiest to read. It's a tradeoff between readability and ease of implementation.

Reply:
- If there is a clash we can live with this. 
- The string does not have to be very descriptive, it is not important in any way. 
- It's much easier to put some string than copy this "rule" here and adjust it and import os.

Reply:
Alright, I guess clashes are indeed not that bad. We'll throttle after 5 occurrences instead of 10 which is okay. If we're not paying too much attention to make the string unique then I agree that having a custom one may be easier than copying this "rule". Let's change it then

Reply:
Good points, change is now pushed